Tests/Tests.hs

```diff
@@ -1,6 +1,6 @@
--- Copyright (c) 2013-2014 PivotCloud, Inc.
+-- Copyright (c) 2013-2015 PivotCloud, Inc.
 --
--- Tests
+-- Main
```

Comment:
Originally it was even requested to additionally put a short module description there. But I can't generate that automatically and I don't want to do it manually by myself for all our code. 

So, to answer your question, you have to ask the folks that came up with our (somewhat verbose) copyright header rules. I don't think that engineering has a lot of influence on it. But if we can convince our legal folks of something simpler (and more standard) I would be very happy.